src/app.js

```diff
@@ -161,6 +155,31 @@ const slotBuilder = (slot) => {
     elWrapper.appendChild(wrap);
 };
 
+const setHighlightStyle = (slotEl, high) => {
+    const highStyle = getHighlightStyle(slotEl);
+    const parentWrapper = slotEl.closest('.ribbon-mobile-ad-wrapper');
+    const currentScrollableElement = parentWrapper === null ? slotEl : parentWrapper;
+    const position = window.getComputedStyle(currentScrollableElement).position;
+
+    if (position === 'fixed') {
+        highStyle.position = 'fixed';
+    } else {
+        highStyle.position = 'absolute';
+    }
+
+    utils.setStyle(high, highStyle);
+};
+
+const scrollTo = (element) => {
+    const parentWrapper = element.closest('.ribbon-mobile-ad-wrapper');
```

Comment:
I think you would need mobile conditional here, this probably would not work on desktop right?

Reply:
unless this parentWrapper exists only on mobile; I would rename the variable like "mobileParentWrapper"

Reply:
@sarahcalande this is only for mobile .ribbon-mobile-ad-wrapper